msl_raptor_params:
  backpack:
    init_cov_diag: [0.3, 0.3, 0.3, 0.0000000005, 0.00000005, 0.00000005, 0.3, 0.3, 0.3, 0.00005, 0.00005, 0.0005]
    process_noise_diag: [0.000002, 0.0000002, 0.0000002, 0.0000001, 0.000000001, 0.000001, 0.000006, 0.00000006, 0.0000006, 0.0000001, 0.000001, 0.0000001]
    meas_noise_diag: [5, 5, 15, 10]
  survivor:
    init_cov_diag: [0.0001, 0.0001, 0.00001, 0.0000000005, 0.00000005, 0.00000005, 0.003, 0.003, 0.003, 0.0005, 0.0005, 0.005]
    process_noise_diag: [0.000002, 0.0000002, 0.0000002, 0.0000001, 0.000000001, 0.000001, 0.000006, 0.00000006, 0.0000006, 0.0000001, 0.000001, 0.0000001]
    meas_noise_diag: [3, 3, 5, 3]
  vent:
    init_cov_diag: [0.5, 0.5, 0.5, 0.0000000005, 0.00000005, 0.00000005, 0.5, 0.5, 0.5, 0.00005, 0.00005, 0.0005]
    process_noise_diag: [0.000002, 0.0000002, 0.0000002, 0.0000001, 0.000000001, 0.000001, 0.000006, 0.00000006, 0.0000006, 0.0000001, 0.000001, 0.0000001]
    meas_noise_diag: [5, 5, 15, 10]
  helmet:
    init_cov_diag: [0.5, 0.5, 0.5, 0.0000000005, 0.00000005, 0.00000005, 0.5, 0.5, 0.5, 0.00005, 0.00005, 0.0005]
    process_noise_diag: [0.000002, 0.0000002, 0.0000002, 0.0000001, 0.000000001, 0.000001, 0.000006, 0.00000006, 0.0000006, 0.0000001, 0.000001, 0.0000001]
    meas_noise_diag: [5, 5, 30, 15]
  rope:
    init_cov_diag: [0.5, 0.5, 0.5, 0.0000000005, 0.00000005, 0.00000005, 0.5, 0.5, 0.5, 0.00005, 0.00005, 0.0005]
    process_noise_diag: [0.000002, 0.0000002, 0.0000002, 0.0000001, 0.000000001, 0.000001, 0.000006, 0.00000006, 0.0000006, 0.0000001, 0.000001, 0.0000001]
    meas_noise_diag: [5, 5, 30, 15]
